functions/controllers/processStaffAugmentationFormSubmit.js

```diff
@@ -36,5 +37,9 @@ async function handlePostRequest (req, res) {
     tags: getMailchimpTags('staff-augmentation')
   })
 
-  res.status(200).end()
+  if (!await isReCAPTCHATokenValid(req.body.recaptchaToken)) {
+    return res.status(403).end()
+  } else {
```

Comment:
This `if` has to be the first statement of the function. We don't want to add the person to MailChimp if they don't provide a valid CAPTCHA token.